DQMOffline/Muon/src/EfficiencyAnalyzer.cc

```diff
@@ -270,10 +270,11 @@ void EfficiencyAnalyzer::analyze(const edm::Event & iEvent,const edm::EventSetup
       h_allProbes_pt->Fill(muon2->pt());
       h_allProbes_eta->Fill(muon2->eta());
       h_allProbes_phi->Fill(muon2->phi());
-      h_allProbes_inner_pt->Fill(muon2->innerTrack()->innerMomentum().Rho());
-      h_allProbes_inner_eta->Fill(muon2->innerTrack()->innerPosition().Eta());
-      h_allProbes_inner_phi->Fill(muon2->innerTrack()->innerPosition().Phi());
-      
+      if(muon2->innerTrack().isNonnull()) {
+          h_allProbes_inner_pt->Fill(muon2->innerTrack()->innerMomentum().Rho());
```

Comment:
this will not help.
Did you check that the crash is gone?

``` innerTrack().extra().isAvailable() ``` looks like a more correct approach

Reply:
@parbol we need to use innerTrack().extra().isAvailable(). isNonnull() doesn't mean that data is available (I know it's confusing if you don't use it each day, but it covers different use cases).